README.md

```diff
@@ -259,6 +259,37 @@ CustomApp.getInitialProps = async (ctx: AppContext) => {
 
 <br />
 
+# ⚗️ CLI (experimental)
+
+You can use `unleash [action] [options]` in you `package.json` `scripts` section, or with:
+
+```sh
+npx @unleash/nextjs
+```
+
+Use [Environment Variables](#environment-variables) (you needed: `UNLEASH_SERVER_API_URL`, `UNLEASH_SERVER_API_TOKEN`), or shorthand versions: `UNLEASH_URL`, `UNLEASH_TOKEN`, `UNLEASH_APPNAME`.
```

Comment:
What do you use environment variables for? I don't understand why this sentence is telling me to do something.

```suggestion
Use [environment variables](#environment-variables) (you need: `UNLEASH_SERVER_API_URL`, `UNLEASH_SERVER_API_TOKEN` or shorthand versions: `UNLEASH_URL`, `UNLEASH_TOKEN`, `UNLEASH_APPNAME`).
```

Reply:
Updated